File: pyrb/Container/1.py
#!/usr/bin/python
import cgi;
import cgitb;cgitb.enable()
print("Content-Type: text/html\n")

name = 'cansmile'
print(type(name))
print(name)
print(type(['cansmile','sunny','gangyi']))
names = ['cansmile','sunny','gangyi']
print(names[0])
print(names[1])
print(names[2])
cansmile = ['evangelist','bucheon',40,True]
cansmile[2] = 'gyeonggi'
print(cansmile)

File: pyrb/Container/1.rb
#!/usr/bin/ruby
puts <<EOS
Content-type: text/html

EOS

names = ['cansmile','gyeonggi',40,true]
puts(names.class)
puts(names[0])
puts(names[1])
puts(names[2])
names[1] = 'bucheon'
print(names)